### PR TITLE
Update OEM building chapter

### PR DESCRIPTION
**Pull request by schaefi**
Redesign building chapter to follow style guide. Issue #323

### CODE REVIEW

File: doc/source/building/build_oem_disk.rst [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
+.. sidebar:: Abstract
 
-   Make sure you have checked out the example image descriptions
-   For details see :ref:`example-descriptions`
+   This page explains how to build an OEM disk image. It contains:
 
-The following example shows how to build and deploy an expandable
-system image based on SUSE Leap
+   * how to build an oem image
+   * how to deploy an oem image
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: 2 * oem -> OEM

File: doc/source/building/build_oem_disk.rst [hunk 2]
```diff
@@ -33,14 +47,22 @@ media.
...
 
-1. Manually deploy the OEM disk image onto the target disk
+1. :ref:`deploy_manually`
+
+   Manually deploy the OEM disk image onto the target disk
 
-2. Boot the OEM installation image and let KIWI's oem installer
+2. :ref:`deploy_from_iso`
+
+   Boot the OEM installation image and let KIWI's oem installer
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: oem -> OEM

File: doc/source/building/build_oem_disk.rst [hunk 3]
```diff
@@ -33,14 +47,22 @@ media.
...
    deploy the OEM disk image from CD/DVD or USB stick onto the target disk
 
-3. PXE boot the target system and let KIWI's oem installer
+3. :ref:`deploy_from_network`
+
+   PXE boot the target system and let KIWI's oem installer
```
Comment (tomschr) [inline doc/source/building/build_oem_disk.rst]: same here: oem -> OEM

**Comment by schaefi** [comment]
Thanks Tom for the review, going to merge it

**Pull request merged**